core/src/CoreTelemetry.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Core
+{
+    internal class CoreTelemetry
```

Comment:
Hmm, how about CoreTelemetryService .... this name makes it a bit clearer that it is a singleton... ?

Reply:
#Resolved